app/internal/page/app.go

```diff
@@ -243,6 +244,23 @@ func testingStart(store sessions.Store, lpaStore LpaStore) http.HandlerFunc {
 			_ = lpaStore.Put(r.Context(), sessionID, lpa)
 		}
 
+		if r.FormValue("howAttorneysAct") != "" {
+			sessionID := base64.StdEncoding.EncodeToString([]byte(session.Values["sub"].(string)))
+			lpa, _ := lpaStore.Get(r.Context(), sessionID)
+
+			switch r.FormValue("howAttorneysAct") {
+			case "jointly":
+				lpa.DecisionsType = "jointly"
+			case "jointly-and-severally":
+				lpa.DecisionsType = "jointly-and-severally"
+			default:
+				lpa.DecisionsType = "mixed"
+				lpa.DecisionsDetails = "some details"
+			}
+
+			_ = lpaStore.Put(r.Context(), sessionID, lpa)
+		}
+
```

Comment:
After the replacement attorney flow is completed we'll work out how we're handling fixtures for cypress - definitely time with this addition.